packages/cli/src/zosfiles/view/ds/Dataset.definition.ts

```diff
@@ -52,6 +53,14 @@ export const DatasetDefinition: ICommandDefinition = {
         {
             description: strings.ACTIONS.DATA_SET.EXAMPLES.EX2,
             options: `"ibmuser.test.loadlib(main)" --binary`
+        },
+        {
+            description: strings.ACTIONS.DATA_SET.EXAMPLES.EX3,
+            options: `"ibmuser.cntl(iefbr14)" --range 0,2`
```

Comment:
As a poor man's UX comment, should we make the range be 1 to n instead of 0 to n - 1, and then convert the user-supplied values into zero-based values for the IBM header?

If other feel that my suggestion is misguided, I am ready to approve the implementation.

Reply:
that's a fair point about UX; on that side i have no strong opinion.  

the implementation is a little tricky though, and so i avoided it.  right now, we just pass through whatever the user input was, without validation or modification.  to alter the UX we'd need to distinguish between `SSS,NNN` and `SSS-NNN`.  in the former, decrement only `SSS` by one; in the latter, decrement `SSS` and `NNN` by one.  Preceding zeros are allowed for both `SSS` and `NNN`, and so are other special case situations, e.g. `--range 1-0` (return the entire file except the first row).  so, if the value is zero, we would not decrement.  

given that it makes the implementation more difficult, i lean towards keeping it as is.  what are your thoughts?

Reply:
I am not a fan of making things more complicated. It frequently adds more edge-case problems. I think we have other commands that retain the characteristics of the underlying zosmf API. I wanted to ensure that we thought about our choices, but I have no objection to retaining the zero-based numbering.